.github/workflows/testing.yml

```diff
@@ -43,6 +43,60 @@ jobs:
       - name: Composer Config
         run: composer config repositories.social vcs $GITHUB_WORKSPACE/tmp/repository
 
+      - name: Clear composer cache
+        run: composer clearcache
+
+      # Install the version of Open Social under test.
+      - name: Composer Install
+        run: composer require goalgorilla/open_social:dev-${{ github.sha }}#${{ github.sha }}
+
+      - name: Set file permissions
+        run: mkdir -p $GITHUB_WORKSPACE/sites/default/files && sudo chmod -R 664 $GITHUB_WORKSPACE/sites/default/files
+
+      - name: PHPUnit
+        run: $GITHUB_WORKSPACE/vendor/bin/phpunit -c $GITHUB_WORKSPACE/html/profiles/contrib/social/phpunit.xml.dist --log-junit $GITHUB_WORKSPACE/test-reports/phpunit.xml
+        env:
+          SIMPLETEST_DB: sqlite://tmp/db.sqlite
+
+  phpunit8:
+    name: PHPUnit on PHP 8.0
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Set default php8.0
+        run: sudo update-alternatives --set php /usr/bin/php8.0
+
+      # We checkout the code in a separate folder since we want to use the
+      # result of the merge rather than the pre-merged code.
+      # A separate path is used to keep our working directory clean.
+      - uses: actions/checkout@v3
+        with:
+          path: ${{ github.workspace }}/tmp/repository
+          # Since composer must clone from this we require all the history.
+          fetch-depth: 0
+
+      - name: Cache Composer dependencies
```

Comment:
does it make sense to still cache all our dependencies if we are going to clear the composer cache anyway? it might be faster to not cache anything and not have to clear it, or will that break somehow?

Reply:
This was an a experimentation step which we need to remove 🙈